vignettes/theoretical_background.Rmd

```diff
@@ -0,0 +1,102 @@
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+link-citations: true
+vignette: >
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Theoretical background}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  message = FALSE,
+  warning = FALSE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+_finalsize_ provides methods to calculate the final proportion of individuals in different age groups infected during an epidemic, as predicted by an age-stratified SIR model. In this vignette we lay out the mathematical concepts behind the functionality available in the package.
+
+## From SIR to final size with homogeous mixing
+
+The dynamics of the proportion of susceptible and infectious individuals over time in the standard susceptible-infectious-recovered (SIR) epidemic model are given by the following ordinary differential equations:
+$$
+\frac{d S(t)}{d t} = -\beta S(t) I(t) \\
+\frac{d I(t)}{d t}  =  \beta S(t) I(t) -\gamma I(t)
+$$
+where $\beta$ is the transmission rate and $\gamma$ is the rate of recovery.
+
+To calculate the proportion infected, we need to derive an expression for the proportion of susceptibles who got infected during the epidemic, i.e. $\phi = 1- S(\infty)/S(0)$. Rather than simulating the model over a very long time period (i.e. $t \rightarrow \infty$), we can instead calculate $dI/dS$ and integrate:
+
+$$
+\frac{d I}{d S} = - 1 + \frac{\gamma}{\beta S} \\
+I(t) = - S(t) + \frac{\gamma}{\beta} \log S(t) + c
+$$
+where $c$ depends on initial conditions. Under the assumption the population is fully susceptible initially (i.e. $S(0)=1$), and $I(t) \rightarrow 0$ as $t \rightarrow \infty$, we obtain the following:
+$$
+\log S(\infty) = - R_0 [1-S(\infty) ] \\
+\mathrm{log}~\phi  -R_0 (1-\phi) = 0 = F(\phi)
+$$
+This equation doesn't have an analytical solution, but can be solved numerically with Newton’s method to find the final epidemic size, i.e. the unique solution in (0,1]:
+
+1. Select an arbitrary initial estimate $x_0$ and tolerance $T$ ;
+2. Solve $\frac{dF}{d \phi} (x_0) \Delta x = −F(x_0) ~ \text{for } ~ \Delta x$ ;
+3. Set $x_{i+1} = x_i + \Delta x$;
+4. Repeat until $x_i$ converges sufficiently (i.e. difference in each step is less than $T$)
+
+## Varying susceptibility
+
+We can extend the above formulation to account for variable susceptibility among individuals, following the methods outlined in [Miller (2012)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3506030/). First, we define $p(x)$ to be the probability density for a randomly chosen individual to be of suceptibility type $x$. If we define the probability an individual of type $x$ to be ultimately infected as $\phi(x)$, then the overall proportion of the population to be infected is defined as follows, depending on whether $p(x)$ is a continuous or discrete distribution:
```

Comment:
Spelling of (2nd) suceptibility -> susceptibility

Reply:
Now fixed